config.c

```diff
@@ -990,21 +990,12 @@ NORETURN
 static void die_bad_number(const char *name, const char *value)
 {
 	const char *error_type = (errno == ERANGE) ?
-		N_("out of range") : N_("invalid unit");
-	const char *bad_numeric = N_("bad numeric config value '%s' for '%s': %s");
+		_("out of range") : _("invalid unit");
+	const char *bad_numeric = _("bad numeric config value '%s' for '%s': %s");
```

Comment:
Maybe split this `N_()` -> `_()` change out into its own commit? It is a bit distracting from the concern(s) covered by the rest of the patch.

Reply:
makes sense, moved to a separate commit with message